lib/src/vector_math/vector2.dart

```diff
@@ -20,14 +20,15 @@ class Vector2 implements Vector {
     result.y = Math.max(a.y, b.y);
   }
 
-  // Interpolate between [min] and [max] with the amount of [a] using a linear
-  // interpolation and set the values to [result].
+  /// Interpolate between [min] and [max] with the amount of [a] using a linear
+  /// interpolation and store the values in [result].
   static void mix(Vector2 min, Vector2 max, double a, Vector2 result) {
     result.x = min.x + a * (max.x - min.x);
     result.y = min.y + a * (max.y - min.y);
   }
 
-
+  /// The components of the vector.
+  Float32List get storage => _storage;
```

Comment:
@kevmoo  Vector expects a public storage getter, at that one is still here. Shouldn't break anything?